src/Products/PluggableAuthService/plugins/tests/test_CookieAuthHelper.py

```diff
@@ -159,8 +162,9 @@ def test_challenge_with_vhm(self):
         self.assertEqual(response.status, 302)
         self.assertEqual(len(response.headers), 3)
         loc = response.headers['Location']
-        self.assertTrue(loc.endswith(quote(actualURL)))
+        self.assertTrue(loc.endswith(quote('/xxx')))
         self.assertFalse(loc.endswith(quote(vhm)))
+        self.assertNotIn(actualURL, loc)
```

Comment:
By skimming the patch it seems to me that `actualURL` can never be there because it is processed by the `quote` function:

I would just write:
```
self.assertEqual(loc, expected_url)
```
to make the test more strict and straightforward to read.

Anyway this is really a minor thing.

Reply:
Good idea